app/src/main/kotlin/com/intive/picover/parties/viewmodel/PartiesViewModel.kt

```diff
@@ -27,8 +27,9 @@ class PartiesViewModel @Inject constructor(
 		loadParties()
 	}
 
-	private fun loadParties() {
+	fun loadParties() {
 		viewModelScope.launch {
+			state.value = Loading
```

Comment:
I am wondering here about the state setting.

Now we are using `init { }` to fetch parties first time, and using loading state at the beginning.
When we want to do a retry, we need to set loading state again, so will be good to have undefined state at the beginning and set loading

But there is a problem with testing, if the state is not a flow and we want to check undefined state, set before init block execution.

I've also realised that we are not using MVI at all, ase we planned to try :D Maybe i can create some proposal of MVI solution here, in other issue #155

Reply:
We were talking about it at the guild meeting and my suggestion during the meeting was: 

if we don't want to add an undefined state we can just add a new method for refreshing data which just invokes another method in VM (yes Marcin we discussed about this method could be redundant 😄), and before invoking this method we can set the state to loading, and thanks to that we can omit doubled loading state.

We can go with doubled loading state exactly like now, or we can go with an undefined state. 

For me whatever you choose, I'm in, because if we rewrite it on MVI probably the problem disappear.